resources/tentative/charts/observable-plot.js

```diff
@@ -0,0 +1,265 @@
+            state,
+            iata,
+            name,
+            city,
+            index, // This will be used to have consistent colors.
+            ...flightsByAirport.get(iata),
+        }));
+        enrichedMostUsedAirports.push({
+            state,
+            iata: "Other",
+            total: total - d3Array.sum(mostUsedAirports, ({ iata }) => flightsByAirport.get(iata)?.total),
+            index: enrichedMostUsedAirports.length,
+        });
+        return enrichedMostUsedAirports;
+    });
+
+    preparedData = {
+        airports,
+        flights,
+        flightsByAirport,
+        byAirport,
+        stateInformationSortedArray,
+        stateSortedArray,
+        statesWithMostFlights,
+        plotData,
+    };
+}
+
+function parseAirportsInformation() {
+    return {
+        airports: csvParse(airportsString),
+        flights: csvParse(flightsString),
+    };
+}
+
+function groupFlightsByAirports(flights) {
+    const flightsByAirport = new Map();
+
+    for (const { origin, destination, count } of flights) {
+        const infoForOriginAirport = flightsByAirport.get(origin) ?? { origin: 0, destination: 0, total: 0 };
+        const intCount = Number(count);
+        if (Number.isNaN(intCount)) {
+            console.error(`Couldn't convert ${count} to number.`);
+            continue;
+        }
+        infoForOriginAirport.origin += intCount;
+        infoForOriginAirport.total += intCount;
+        flightsByAirport.set(origin, infoForOriginAirport);
+        const infoForDestinationAirport = flightsByAirport.get(destination) ?? { origin: 0, destination: 0, total: 0 };
+        infoForDestinationAirport.destination += intCount;
+        infoForDestinationAirport.total += intCount;
+        flightsByAirport.set(destination, infoForDestinationAirport);
+    }
+
+    return flightsByAirport;
+}
+
+function isReady() {
+    return !!preparedData;
+}
+
+function addStackedBars() {
+    if (!isReady())
+        throw new Error("Please prepare the data first.");
+
+    const options = {
+        ...DEFAULT_OPTIONS,
+        color: { type: "categorical" },
+        x: {
+            domain: preparedData.stateSortedArray,
+        },
+        y: { grid: true, tickFormat: "~s" },
+        marks: [
+            // stacked bars
+            Plot.barY(preparedData.plotData, {
+                x: "state",
+                y: "total",
+                fill: "index",
+                title: (d) => `${d.iata === "Other" ? "Other" : `${d.name}, ${d.city} (${d.iata})`}\n${d3Format(",")(d.total)} flights`,
+            }),
+            // labels
+            Plot.text(preparedData.stateInformationSortedArray, { x: "state", y: "total", text: (d) => d3Format(".2~s")(d.total), dy: -10 }),
+            // horizontal bottom line
+            Plot.ruleY([0]),
+        ],
+    };
+    ROOT.append(Plot.plot(options));
+}
+
+// Note that this function is currently commented out in the benchmark. We may
```

Comment:
I think we should just delete it. It's easy enough to add back if we see the need for it.

Reply:
I don't find that "easy" myself :D 
but otherwise yeah, I wanted to keep it for the sake of discussing about including it, and I'm happy to remove it if we agree that we don't need it.